File: curso-python/ex079.py
# criando a lista que guarda os valores
valores = []
# fazendo um enquanto infinito
while True:
    novos_valores = int(input("digite um valor: "))

    # verificar se 3já tem um valor igual
    if novos_valores not in valores:
        valores.append(novos_valores)
        valores.sort()
        print("valor adicionado com sucesso...")
    else:
        print('valor duplicado! Não vou adicionar...')

    # para verificar se o usuário quer ou não continuar
    resp = ' '
    while resp not in 'SNsn':
        resp = input("quer continuar [S/N]: ").strip().upper()
    
    if resp in 'Nn':
        print("saindo do programa...")
        break

print(valores)

File: javascript-exercicios/ex079.js
const prompt = require("prompt-sync")();

// criando a lista que guarda os valores
let valores = [];

while(true){
    let novos_valores = Number.parseInt(prompt("digite um valor: "));

    if (!valores.includes(novos_valores)){
        valores.push(novos_valores);
        valores.sort();
        console.log("valor adicionado com sucesso... ");
    }else{
        console.log("valor duplicado! não vou adicionar");
    }

    // para verificar se o usuário quer ou não continuar
    let resp = ' ';
    while (resp != 'S' && resp != 'N'){
        resp = prompt("quer continuar[S/N]: ").trim().toUpperCase();
    }
    if (resp == "N"){
        console.log("saindo do programa... ")
        break;
    }
    
}

console.log(valores);